nautobot/dcim/api/urls.py

```diff
@@ -70,6 +70,11 @@
 # Device Redundancy Group
 router.register("device-redundancy-groups", views.DeviceRedundancyGroupViewSet)
 
+# Software images
+router.register("software-images", views.SoftwareImageViewSet)
+router.register("software-versions", views.SoftwareVersionViewSet)
+router.register("device-type-to-software-image", views.DeviceTypeToSoftwareImageViewSet)
```

Comment:
Given our convention of pluralizing the URL path strings, should this be something like `device-types-to-software-images` instead?

Reply:
In IPAM we have `/api/ipam/ip-address-to-interface/` but then we have `/api/dcim/interface-redundancy-group-associations/`. I think either the double plural or the `associations` makes it more apparent that you're dealing with an m2m. What do you think?

Reply:
I think `ip-address-to-interface` also feels "wrong" to me. I'd vote for the double plural.

Reply:
And I forgot to fix this before merging 😦 
I opened #5190 to fix this